fixtures/src/components/uncontrolled/decimal.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Decimal from 'carbon-react/lib/__experimental__/components/decimal';
+import { LogConsumer } from '../log';
+
+const UncontrolledDecimal = () => {
+  return (
+    <LogConsumer>
+      {(log) => {
+        const onChange = e => log(e, { method: 'onChange' });
+        const onBlur = e => log(e, { method: 'onBlur' });
+        return (
+          <React.Fragment>
+            <div id='uncontrolled_decimal'>
+              <h1>Uncontrolled Decimal</h1>
+              <ul>
+                <li>onChange handler should update the log when the value is changed, e.target.value should be an array
+                  that contains the &#123; formattedValue, rawValue &#125; the Date input&apos;s
+                </li>
+                <li>onBlur handler should update the log when the decimal is blurred, e.target.value should be the
+                  formatted value
+                </li>
+                <li>decimal has props value, name, id which should be reflected in both events</li>
```

Comment:
There's no `value` prop being passed to `<Decimal>` here.
```suggestion
                <li>decimal has props name, id which should be reflected in both events</li>
```

Reply:
Thanks for fixing this in 68630c9 :white_check_mark: